services/comment.js

```diff
@@ -0,0 +1,34 @@
+const Comment = require('~/models/comment')
+const Cooperation = require('~/models/cooperation')
+const { createForbiddenError } = require('~/utils/errorsHelper')
+
+const commentService = {
+  createComment: async (body) => {
+    const { text, author, authorRole, cooperationId } = body

```

Comment:
shouldn't it be 'data' instead of 'body' like in other services?

Reply:
changed